packages/allure-jest/test/fixtures/attachments.test.js

```diff
@@ -1,3 +1,7 @@
 it("json", () => {
   allure.attachment(JSON.stringify({ foo: "bar" }), "application/json");
 });
+
+it("name", () => {
```

Comment:
`it("with name", () => {`, now it looks like we have method called `name`, not an ability to set the property

Reply:
done